sale_order_digitized_signature/readme/CONTRIBUTORS.rst

```diff
@@ -1,4 +1,4 @@
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
-* `Tecnativa <https://www.tecnatica.com>`_:
-
-  * Vicent Cubells
+* Vicent Cubells <vicent.cubells@tecnativa.com>
```

Comment:
Why are you removing the previous form. We choose that one on purpose for showing our company URL.

Reply:
@pedrobaeza My understanding is that individuals go into contributors and companies go into credits... Am I wrong?

Reply:
The problem is that authors part doesn't have the ability to put an URL, and as we don't put now the company url in the manifest, this is the authorized way of do it.

Reply:
@pedrobaeza ok